xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/layouts/table/LayoutTableHeaderNames.vue

```diff
@@ -58,45 +58,49 @@
       v-show="logic.isPropertyVisible(property.id)"
     >
       <!-- Wrapper for the column header -->
-      <div
-        class="column-name"
-        @click="sort(property)"
-      >
+      <div class="column-name">
+        <!-- Property Name -->
+        <button 
+          class="property-name"
+          :title="$t('livedata.action.sort.hint')"
+          @click="sort(property)"
+        >
+          {{ property.name }}
+          <!--
+            Sort icon
+            Only show the icon for the first-level sort property
+          -->
+          <XWikiIcon
+            v-if="logic.isPropertySortable(property.id)"
+            :icon-descriptor="{name: isFirstSortLevel(property) && firstSortLevel.descending? 'caret-down': 'caret-up'}"
+            :class="['sort-icon',  isFirstSortLevel(property)? 'sorted': '']"/>
+        </button>
+        
         <!--
           Specify the handle to drag properties.
-          Use the stop propagation modifier on click event
-          to prevent sorting the column unintentionally.
         -->
-        <div
-          role="presentation"
-          class="handle"
+        <button
+          class="handle btn btn-xs btn-default"
           :title="$t('livedata.action.reorder.hint')"
-          @click.stop
+          @keydown.left="keyboardDragNDropLeft($event)"
+          @keydown.right="keyboardDragNDropRight($event)"
         >
-          <XWikiIcon :icon-descriptor="{name: 'text_align_justify'}"/>
-        </div>
-        <!-- Property Name -->
-        <span class="property-name">{{ property.name }}</span>
+          <XWikiIcon :icon-descriptor="{name: 'right'}"/>
```

Comment:
I proposed introducing the arrows icon as a part of https://github.com/xwiki/xwiki-platform/pull/2593 .
I proposed to name it `reposition` (since `move` was already taken for another definition of move).
I updated this PR to match these changes in https://github.com/xwiki/xwiki-platform/pull/2475/commits/2aae176b07f6b78d2cb95f73237eccab0c9ee512